src/System.Diagnostics.Process/src/System/Diagnostics/Process.WinRT.cs

```diff
@@ -13,18 +13,18 @@ private bool StartCore(ProcessStartInfo startInfo)
                 : StartWithCreateProcess(startInfo);
         }
 
-        private string GetMainWindowTitle() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private string GetMainWindowTitle() => string.Empty;
 
-        public IntPtr MainWindowHandle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public IntPtr MainWindowHandle => IntPtr.Zero;
 
-        private bool CloseMainWindowCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool CloseMainWindowCore() => false;
 
-        public string MainWindowTitle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public string MainWindowTitle => string.Empty;
 
-        private bool IsRespondingCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool IsRespondingCore() => true;
 
-        public bool Responding => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public bool Responding => true;
```

Comment:
Why do all these have both `Responding` and `IsRespondingCore` ? Shouldn't Process.cs be the only one exposing `Responding` and it calls to `IsRespondingCore` in all the partial classes?